modules/sql.js

```diff
@@ -100,6 +105,7 @@ __private.query = function (action, config, cb) {
 
 		try {
 			sql = jsonSql.build(extend({}, config, defaultConfig));
+			library.logger.trace("sql.query: ", sql);
```

Comment:
What do you mean? Postgres does not accept single quotes in the column names enumeration